code/src/java/pcgen/core/AbilityUtilities.java

```diff
@@ -113,7 +113,6 @@ public static String removeChoicesFromName(String name)
 	 * @throws GroupingMismatchException If there are mismatched brackets
 	 */
 	public static String getUndecoratedName(final String name, final Collection<String> specifics)
-		throws GroupingMismatchException
```

Comment:
Now inconsistent with the docs of the method 3 lines up...

Reply:
It is not inconsistent. Instead, the documentation serves the role of documentation.
See https://pmd.github.io/latest/pmd_rules_java_design.html#avoiduncheckedexceptionsinsignatures for the reasoning.